@xen-orchestra/xva-generator/importVdi.mjs

```diff
@@ -0,0 +1,31 @@
+import { isNotEmptyRef } from './_isNotEmptyRef.mjs'
+import { importVm } from './importVm.mjs'
+
+export async function importVdi(vdi, vhd, xapi, sr) {
+  // create a fake VM
+  const vmRef = await importVm(
+    {
+      name_label: `[xva-disp-import]${vdi.name_label}`,
+      memory: 1024 * 1024 * 32,
+      nCpus: 1,
+      firmware: 'bios',
+      vdis: [vdi],
+      vhds: [vhd],
+    },
+    xapi,
+    sr
+  )
+  // ensure VM is laoded
+  await xapi.getRecord('VM', vmRef)
```

Comment:
What's the point of this?

This will fail if the VM was yet imported or it will return its ref, it will not wait.